### ISSUE TITLE
Shopping site Etsy doesn't work on normal and private tabs but it does on brave-tor-window

### ISSUE THREAD

**Issue by NumDeP** [brave/brave-browser#12360]
Have you searched for similar issues? Yes.

## Description 
The dot com and dot com/uk site of Etsy doesn't work. 

## Steps to Reproduce
Open a normal or private tab and enter etsy.com or etsy.com/uk

## Actual result:
```
This page isn’t working
If the problem continues, contact the site owner.
HTTP ERROR 429
``` 

## Reproduces how often:
Well it's not going off.

## Brave version (brave://version info)
Version 1.15.76 Chromium: 86.0.4240.111 (Official Build) (64-bit) (openSuse)

The issue actually occurred before the latest release, I think maybe even as far as a couple of months. 
 
- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? Didn't try.
- Is the issue reproducible on the latest version of Chrome? Must be a Brave specific issue.

## Miscellaneous Information:
I reported this as broken site not too long ago maybe even a month ago, I'm surprised it wasn't seen to. I wonder how you receive a broken site on your end and how and when you decide to fix it.

**Comment by rebron** [comment]
@NumDeP I'm not able to reproduce this on Windows, macOS, or Linux. I'm on the latest and tried both etsy.com and etsy.coom/uk

Brave | 1.16.68 Chromium: 86.0.4240.111 (Official Build) (x86_64)
-- | --
Revision | b8c36128a06ebad76af51591bfec980224db5522-refs/branch-heads/4240@{#1290}
OS | macOS Version 10.16 (Build 20A5395g)

Can you try again on your end?

**Comment by ryanbr** [comment]
Tested with `https://www.etsy.com/nz/listing/793939351/personalised-bridesmaid-pjs-wedding-pjs?ref=hp_prn&pro=1`

Page opens, is this still an issue @NumDeP  ?

**Comment by NumDeP** [comment]
Yep. `Version 1.16.72 Chromium: 86.0.4240.183 (Official Build) (64-bit) (openSuse)`

![Screenshot_20201104_171334](https://user-images.githubusercontent.com/31957540/98144992-9065a000-1ec2-11eb-9741-cb9aedb12109.png)

**Comment by NumDeP** [comment]
Can I ask, is it possible for some Linux based bugs appearing in one OS to not be an issue in other Linux distros or should every single Linux based bug technically appear in every Linux distro? 

The reason I ask is because I'm not sure but I think rebron tested it on Linux Mint without an issue but I can't in openSuse even on `Version 1.17.73 Chromium: 87.0.4280.67 (Official Build) (64-bit)` and I also ran into another issue related to Full Screen mode (F11) which hasn't been working for over a year for me but I noticed it does work for some other in other Linux distros but is slightly buggy.

**Comment by NumDeP** [comment]
@ryanbr @rebron  are you aware this still doesn't work? Just making sure as I think I'm on the latest version and it's still not working for some reason.

**Comment by ryanbr** [comment]
I did notice an error message (similar to yours) but was related on the VPN I was using. Do you use VPN? (or try a different location) @NumDeP

**Comment by NumDeP** [comment]
@ryanbr I did just notice that having just mentioned it. One server would work but strangely another server though in the same location wouldn't. 

Does this mean it's not an issue for Brave to fix?

**Comment by rebron** [comment]
Hi @NumDeP Are you still having this issue? Our latest is 1.20.103. There is definitely cases where we have issues in one distro and not the other. And this site is still working fine for me on Linux Mint. This isn't extension related I hope.

**Comment by rebron** [comment]
@srirambv When you get a chance can you see if you're running into this issue on etsy.com in Linux?

**Comment by plevacon** [comment]
Not sure if this helps anyone, but I as just getting the same issue and tried various shield setting to no avail. 
Tried turning off my VPN and/or changing my location and both resulted in Etsy loading.

**Comment by ryanbr** [comment]
@plevacon  Seems to be related to a VPN locaton, Like I suspect previously. Not much we can do here 

I just rechecked etsy and worked fine from NZ and (US IP), but no doubt if I reconnected again to the VPN the results maybe different. Also cc; @NumDeP

**Comment by NumDeP** [comment]
Yep, still facing the same issue on `Version 1.21.74 Chromium: 89.0.4389.72 (Official Build) (64-bit)` This time quite a number of IPs from UK don't work but within another park it actually does. I don't know whether this is something related to their infrastructure whereby similar to just-eat they're simply blocking VPN requests. Though if that's that case why do other servers in and outside the country work?

**Comment by ryanbr** [comment]
Regarding the VPN, The only thing I can recommend. Take note of your local VPN IP, the time/date and the site you visit and just report the issue to your VPN provider. They'll see if its related to the ip's or the datacenter.

I have 3 different VPN providers, some work well, sometimes it breaks. For example, one provider (will rename unnamed) if connecting to a US IP will occasionally break (page will hang and not load) on amazon pages and some amazon hosted pages like IMDB for example.
